src/private/fnft__nsev_slow_testcases.c

```diff
@@ -0,0 +1,664 @@
+/*
+ * This file is part of FNFT.
+ *
+ * FNFT is free software; you can redistribute it and/or
+ * modify it under the terms of the version 2 of the GNU General
+ * Public License as published by the Free Software Foundation.
+ *
+ * FNFT is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Contributors:
+ * Shrinivas Chimmalgi (TU Delft) 2019-2020.
+ */
+#define FNFT_ENABLE_SHORT_NAMES
+
+#include "fnft__errwarn.h"
+#include "fnft__misc.h" // for misc_sech
+#include "fnft__nsev_slow_testcases.h"
+#include "fnft__nse_discretization.h" // for nse_discretization_degree
+#ifdef DEBUG
+#include <stdio.h>
+#include <string.h> // for memcpy
+#endif
+
+INT nsev_slow_testcases(nsev_slow_testcases_t tc, const UINT D,
```

Comment:
There seems to be a lot of overlap with `nsev_testcases`. Why not integrate these new test cases there instead (e.g., `nsev_testcases_SECH_FOCUSING2` instead of `nsev_slow_testcases_SECH_FOCUSING`)? That should reduce the overlap.

Also, only one `compare_nfs` function is needed I think.

`nsev_slow_testcases_test_fnft` could be renamed to `nsev_testcases_test_fnft_slow` and moved into nsev_testcases.c as well. This file could then be removed (that would also be consistent with the naming 'nsev_testcases`, which indicates testcases for the NSE with vanishing boundaries).

Reply:
Hmm yes that can be done.